.github/workflows/cran-check.yml

```diff
@@ -0,0 +1,89 @@
+# this action runs the same checks on the repository that CRAN does:
+#     1. windows container with latest R release
+#     2. linux container with latest R release
+#     3. linux container with next R release
+# to offer a preview of whether changes to the package will be
+# accepted by CRAN. Testing on Mac is not done because there is not
+# udunits2 library for Mac, as far as I can tell. 
+# It also runs the tests that are too slow for CRAN. These extra tests
+# are in the tests/extra-tests directory.
+# it is derived from the check-standard action from the r-lib repository,
+# which can be seen here:
+# https://github.com/r-lib/actions/blob/master/examples/check-standard.yaml
+name: CRAN-check
+on: pull_request
+
+jobs:
+  check:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: windows-latest, r: 'release'}
```

Comment:
Instead of latest could we specify the version here? I just don't want this to break in the future due to a version change.